.ci/daily/Jenkinsfile

```
@@ -15,7 +15,7 @@ pipeline {
   }
   parameters {
     booleanParam name: 'UPSTREAM_PR', defaultValue: false, description: 'Is the current build triggered in the scope of an upstream pull request'
-    string name: 'UPSTREAM_PROJECT_NAME', defaultValue: 'cambpm-ce/cambpm-main/master'
+    string name: 'UPSTREAM_PROJECT_NAME', defaultValue: '7.XX/cambpm-ce/cambpm-main/master'
```

Should we add a description here that the `UPSTREAM_PROJECT_NAME` is passed by the main CE pipeline build? We can also add that this property is not used for standalone runs.

This is just a suggestion, to make this property less confusing for anyone that looks at it for the first time, or tries to run a standalone build from the CI.

```suggestion
    string name: 'UPSTREAM_PROJECT_NAME', defaultValue: '7.XX/cambpm-ce/cambpm-main/master', description: 'Not used on standalone runs. This property is passed by the upstream build that triggers this one.'
```